### PR TITLE
Improve Show selected languages in library header as a subtitle

### PR DESCRIPTION
**Pull request by water2015sit**
Create a wrapper class for divider item to include subtitle information.
Get selected languages and convert them to the subtitle.
Add subtitle text view in the library divider layout file.
Extract two hardcoded strings to string res file.

Screenshots/GIF for the change: 
Screenshot_20190407-130551
Screenshot_20190407-130621
Screenshot_20190407-163442

### CODE REVIEW

**Comment by water2015sit** [comment]
@mhutti1 @siddharth2010 hi, this is Runzhi, please review my pr, thank you!

File: app/src/main/java/org/kiwix/kiwixmobile/library/LibraryAdapter.java [hunk 1]
```diff
@@ -328,6 +336,8 @@ private void saveNetworkLanguages() {
   private static class ViewHolder {
 
     TextView title;
+
```
Comment (iadeelzafar) [inline app/src/main/java/org/kiwix/kiwixmobile/library/LibraryAdapter.java]: This extra line can be removed.

File: app/src/main/java/org/kiwix/kiwixmobile/library/LibraryAdapter.java [hunk 2]
```diff
@@ -370,10 +380,24 @@ protected FilterResults performFiltering(CharSequence s) {
...
             .blockingGet();
 
-        listItems.add(new ListItem(context.getResources().getString(R.string.your_languages),
+        // Get all selected displayed languages from books.
+        HashSet<String> selectedLanguageSet = new HashSet<>();
+        for (Book book : selectedLanguages) {
+          selectedLanguageSet.add(bookUtils.getLanguage(book.getLanguage()));
+        }
+        
+        listItems.add(
```
Comment (iadeelzafar) [inline app/src/main/java/org/kiwix/kiwixmobile/library/LibraryAdapter.java]: Did you refactored the code using Square codestyle?

Comment (water2015sit) [reply app/src/main/java/org/kiwix/kiwixmobile/library/LibraryAdapter.java]: I just reformat the code.

File: app/src/main/java/org/kiwix/kiwixmobile/library/LibraryAdapter.java (hunk 1 above)
Comment (water2015sit) [reply app/src/main/java/org/kiwix/kiwixmobile/library/LibraryAdapter.java]: Done!

**Comment by mhutti1** [comment]
This is changing in 2.5 I don't think its needed yet.

**Pull request closed without merging**